packages/git-proxy-cli/git-proxy-cookie

```diff
@@ -0,0 +1 @@
+["connect.sid=s%3A7JixoS9s9h-iVPSdaMXuYo4QU3nU25I8.iDVsAicZCFXnMFQr%2Bz7Jxv%2Fv0poOuR1KM%2Fe%2Bcz2cbgo; Path=/; HttpOnly"]
```

Comment:
Rather than hardcoding a magic string, should this not be configurable? Let me know if I am missing the point or misunderstanding.